lib/utils/include/utils/graph/views.h

```diff
@@ -230,6 +231,88 @@ struct OpenMultiDiSubgraphView : public IOpenMultiDiGraphView {
   IOpenMultiDiGraphView const &g;
 };
 
+template <typename NodeLabel,
+          typename EdgeLabel>
+struct LabelledMultiDiSubgraphView {
+public:
+  LabelledMultiDiSubgraphView() = delete;
+  template<typename InputLabel, typename OutputLabel>
+  explicit LabelledMultiDiSubgraphView(
+    ILabelledOpenMultiDiGraphView<NodeLabel, EdgeLabel, InputLabel, OutputLabel> const &,
```

Comment:
Should probably be refactored to use `maybe_owned_ref`, iirc this is the most recent interface